VERSION.txt

```diff
@@ -1 +1 @@
-3.0.0.99
```

Comment:
NOTE TO SELF: change this back and re-generate `R-package/configure` once this PR is merged